### ISSUE TITLE
the link to tensorflow version "dispflownet-tf" is unsupervised?

### ISSUE THREAD

**Issue by xadxxadx** [CVLAB-Unibo/Unsupervised-Adaptation-for-Deep-Stereo#9]
hi, I prefer to use tensorflow, but it look like supervised version? because it need ground true when training?

thank you

**Comment by AlessioTonioni** [comment]
Hi,
Both the normal supervised version and the ICCV adaptation are implemented in the tensorflow version, however, to use the adaptation loss you will need to provide disparity map and confidence computed externally. 
To generate the confidence map for your disparity estimation using CCNN please refer to https://github.com/fabiotosi92/CCNN-Tensorflow

**Comment by xadxxadx** [comment]
hi, thank you very much,
i took a look to your link, Yes, it generate confidence.(it work)
but the link [https://github.com/AlessioTonioni/dispflownet-tf](url)

"path_left_frame;peth_right_frame;path_ground_truth", for UA "path_left_frame;peth_right_frame;path_ground_truth;path_confidence"

is that mean I have to train the mode with left+right+ground True image?

thanks

**Comment by AlessioTonioni** [comment]
In the unsupervised version "path_ground_truth" is the path to the disparity map obtained with classic stereo algorithms (SGM or CENSUS on the paper) over which you compute the confidence measure. 
So basically for unsupervised adaptation using SGM you will need a training file were each line is:

"path_left_frame;path_right_frame;disparity_with_SGM;confidence_over_SGM_disparity"

Thank you for pointing out the ambiguity I will fix the README!
